Cargo.toml

```diff
@@ -50,6 +50,10 @@ derive_more = { version = "0.99", default-features = false, features = [
 ] }
 itertools = "0.12"
 serde = { version = "1.0", features = ["derive"] }
+typetag = "0.2"
```

Comment:
This won't work on WASM: we need to find an alternate solution.

Reply:
I checked and it seems there is a crate called `bevy_serde_project` that reimplements `typetag` for WASM, but I'm not sure if it's really usable, as it's still in version 0.1